src/content/content_recommender.py

```diff
@@ -1,18 +1,34 @@
+import numpy as np
+from sklearn.feature_extraction.text import TfidfVectorizer
+from sklearn.metrics.pairwise import cosine_similarity
 class ContentRecommender:
 
     def __init__(self):
-        pass
+        self.word_tfidf = TfidfVectorizer(ngram_range=(1, 1), analyzer='word', sublinear_tf=True, max_df=0.5, stop_words='english')
 
 #TODO - Fit - form a vocab using training events(all valid events).
 #TODO - Ideas for vocab (1 gram, 2 gram or ngram).
     def train(self, training_events):
         ## input : dict([user, list of events])
         ## output : dict([user, tfidf - vector])
+        # Assumed training events to be a list of event_info objects
+        training_events = np.array([event.description for event in training_events])
```

Comment:
actually, I was bit confused. So just kept writing code. Here training events represent all the events like before a timestamp(for all users).
I mean how do you think we will supply data to this function.?
And what are those classes in data-model for ?